changelog.d/11406.bugfix

```diff
@@ -0,0 +1 @@
+Removes unnecessary use of jquery on the recaptcha page from `synapse/synapse/res/templates/recaptcha.html`.
```

Comment:
I don't think this will mean much to the average synapse admin. It would be better to say where this page is used.

I believe it's the fallback page for clients which don't support recaptcha themselves? Have you been able to test it, at all?

Reply:
@richvdh I ran the local server but I'm unable to test it, I'm unable to find the correct path of the page.
![error](https://user-images.githubusercontent.com/76884959/143287470-77fa2582-d1f3-4bd0-b7cf-48817705f62b.png)

This is the url `http://localhost:8080/_matrix/static/` what should be the path to append if I want to test this feature :+1: )

Reply:
http://localhost:8008/_matrix/client/v3/auth/m.login.recaptcha/fallback/web?session=abc is the location of the recaptcha fallback page, which should load the contents of the configured `recaptcha.html` template.

This was a discussion on how to test this manually locally in #synapse-dev here: https://matrix.to/#/!XaqDhxuTIlvldquJaV:matrix.org/$xWMp0Ah_BKfWz9IxE7uUP2Tpzanem6hdDJjRaxsXGNI?via=matrix.org&via=vector.modular.im&via=envs.net